content/php/concepts/math-functions/terms/pi/pi.md

```diff
@@ -0,0 +1,47 @@
+---
+Title: 'pi()'
+Description: 'Returns the value of PI.'
+Subjects:
+  - 'Web Development'
+  - 'Code Foundations'
+Tags:
+  - 'Methods'
+  - 'Arithmetic'
+  - 'Numbers'
+  - 'Functions'
+CatalogContent:
+  - 'learn-php'
+  - 'paths/computer-science'
+---
+
+The **`pi()`** function returns the value of PI.
+
+## Syntax
+
+```pseudo
+pi();
+```
+
+The `pi()` function has no parameters. It returns the value of PI as float. `M_PI`, being a constant, also returns the same value.
+
+## Example
+
+```php
+<?php
+  echo pi();
+?>
+```
+
+This will output:
+
+```shell
+// 3.1415926535898
+```
+
+## Codebyte Example
```

Comment:
The codebyte example shouldn't be the same as the prior example. Either eliminate the codebyte or use `pi()` in some sort of calculation.

Reply:
noted on this

will do further edits